### PR TITLE
Update docs with API version in url

### PR DESCRIPTION
**Pull request by abdullah248**
## Description
Add REST API version to customer facing docs and update examples

## Related issues
[AB#82661](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/82661)

## Testing
Describe how this change was tested.

### CODE REVIEW

File: docs/api-versioning.md [hunk 1]
```diff
@@ -4,32 +4,45 @@ This guide gives an overview of the API version policies for DICOM Server.
 
 All versions of the DICOM APIs will always conform to the DICOMweb™ Standard specifications, but versions may expose different APIs based on our [conformance statment](https://github.com/microsoft/dicom-server/blob/main/docs/resources/conformance-statement.md).
```
Comment (alexandersoy) [inline docs/api-versioning.md:5]: typo at statement...It is written as statment

File: docs/api-versioning.md [hunk 2]
```diff
@@ -4,32 +4,45 @@ This guide gives an overview of the API version policies for DICOM Server.
...
 
 ### Prerelease versions
 
 An API version with the label "prerelease" indicates that the version is not ready for production, and should only be used in testing environments. These endpoints may experience breaking changes without notice.
 
-### Breaking changes
+### How Versions Are Incremented
 
-We will increment the API version number for any breaking changes to the API.
+We currently only increment the major version when ever there is a breaking change which is considered to be not backwards compatible. All minor versions are implied to be `0`. All versions are in the format `Major.0`
```
Comment (alexandersoy) [inline docs/api-versioning.md:30]: whenever

File: docs/tutorials/use-dicom-web-standard-apis-with-c#.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ Once you have deployed an instance of the Medical Imaging Server for DICOM, retr
 1. Sign into the [Azure Portal](https://portal.azure.com/).
 1. Search for **App Services** and select your Medical Imaging Server for DICOM App Service.
 1. Copy the **URL** of your App Service.
+1. Append the version you would like to use to the end of your service url (ex. `https://<service_url>/v<version>/`) and use this as the url for your DICOM service in all the following examples. For more information on versioning visit the [Api Versioning Documentation](../api-versioning.md).
```
Comment (smithago) [inline docs/tutorials/use-dicom-web-standard-apis-with-c#.md:24]: In line #39 IDicomWebClient client = new DicomWebClient(httpClient);

We would recommend to pass the supported version they want to use.

File: docs/tutorials/use-dicom-web-standard-apis-with-curl.md [hunk 4]
```diff
@@ -80,7 +81,7 @@ _Details:_
  > * `Content-Type: multipart/related; boundary=ABCD1234`
  > * `Content-Type: multipart/related`
 
-`curl --request POST "http://{service-name}.azurewebsites.net/studies/1.2.826.0.1.3680043.8.498.13230779778012324449356534479549187420" --header "Accept: application/dicom+json" --header "Content-Type: multipart/related; type=\"application/dicom\"" --form "file1=@{path-to-dicoms}/blue-circle.dcm;type=application/dicom"`
+`curl --request POST "http://{service-name}.azurewebsites.net/v{version}/studies/1.2.826.0.1.3680043.8.498.13230779778012324449356534479549187420" --header "Accept: application/dicom+json" --header "Content-Type: multipart/related; type=\"application/dicom\"" --form "file1=@{path-to-dicoms}/blue-circle.dcm;type=application/dicom"`
```
Comment (smithago) [inline docs/tutorials/use-dicom-web-standard-apis-with-curl.md:84]: Is there a simpler way to specify the base url in one place

Like start the doc with base url like
{Web service url}/{version}

and reference the base url everywhere?

File: docs/resources/Conformance-as-Postman.postman_collection.json [hunk 5]
```diff
@@ -54,11 +53,12 @@
 					}
 				},
 				"url": {
-					"raw": "{{baseUrl}}/studies",
+					"raw": "{{baseUrl}}/v{{version}}/studies",
```
Comment (smithago) [inline docs/resources/Conformance-as-Postman.postman_collection.json:56]: same here

**Review by smithago — APPROVED** [review]

File: docs/tutorials/use-dicom-web-standard-apis-with-curl.md [hunk 6]
```diff
@@ -80,7 +81,7 @@ _Details:_
  > * `Content-Type: multipart/related; boundary=ABCD1234`
  > * `Content-Type: multipart/related`
```
Comment (alexandersoy) [inline docs/tutorials/use-dicom-web-standard-apis-with-curl.md:82]: Do not we need to update headers as well for versioning?

Comment (alexandersoy) [reply docs/tutorials/use-dicom-web-standard-apis-with-curl.md:82]: such as: 
* Path: ../studies
* Method: POST
* Headers:
    * Accept: application/dicom+json
    * Content-Type: multipart/related; type="application/dicom"
    * Authorization: Bearer {token value}
    * **Api-supported-versions: v{version}**

Comment (abdullah248) [reply docs/tutorials/use-dicom-web-standard-apis-with-curl.md:82]: We dont specify the version as header (only specified in the path url). I think you may be referring in the response headers, we specify if the version the user is using is deprecated or not.

File: docs/resources/Conformance-as-Postman.postman_collection.json (hunk 5 above)
Comment (abdullah248) [reply docs/resources/Conformance-as-Postman.postman_collection.json]: Added  a note to the instructions for using postman and changed the default of baseUrl to include the version

File: docs/tutorials/use-dicom-web-standard-apis-with-curl.md (hunk 4 above)
Comment (abdullah248) [reply docs/tutorials/use-dicom-web-standard-apis-with-curl.md]: Updated, thanks!

File: docs/tutorials/use-dicom-web-standard-apis-with-c#.md (hunk 3 above)
Comment (abdullah248) [reply docs/tutorials/use-dicom-web-standard-apis-with-c#.md]: Updated

**Pull request merged**